src/main/java/io/strimzi/kafka/bridge/quarkus/RestBridge.java

```diff
@@ -548,7 +565,53 @@ private boolean isReady() {
         return this.isReady;
     }
 
-    private void startInactiveConsumerDeletionTimer(Long timeout) {
-        // TODO: to be used a Quarkus timer for scheduling the task
+    private void startInactiveConsumerDeletionTimer(long timeout) throws SchedulerException {
+        long timeInMs = timeout * 1000L;
+        JobDetail job = JobBuilder.newJob(InactiveConsumerDeletionJob.class).build();
+        Trigger trigger = TriggerBuilder.newTrigger()
+                .withIdentity("inactiveConsumerDeletion", "staleConsumers")
+                .startNow()
+                .withSchedule(
+                        SimpleScheduleBuilder.simpleSchedule()
+                                .withIntervalInMilliseconds(timeInMs / 2)
+                                .repeatForever())
+                .build();
+        quartz.scheduleJob(job, trigger);
+    }
+
+    /**
+     * Looks up for the stale consumers and remove them if they have passed the timeout
+     */
+    void performTask() {
+        long timeoutInMs = httpConfig.timeoutSeconds() * 1000L;
+
+        log.debugf("Looking for stale consumers in %s entries", timestampMap.size());
+        Iterator<Map.Entry<ConsumerInstanceId, Long>> it = timestampMap.entrySet().iterator();
+        while (it.hasNext()) {
+            Map.Entry<ConsumerInstanceId, Long> item = it.next();
+            if (item.getValue() + timeoutInMs < System.currentTimeMillis()) {
+                RestSinkBridgeEndpoint<byte[], byte[]> deleteSinkEndpoint = httpBridgeContext.getHttpSinkEndpoints().get(item.getKey());
+                if (deleteSinkEndpoint != null) {
+                    deleteSinkEndpoint.close();
+                    httpBridgeContext.getHttpSinkEndpoints().remove(item.getKey());
+                    log.debugf("Consumer %s deleted after inactivity timeout.", item.getKey());
+                    timestampMap.remove(item.getKey());
+                }
+            }
+        }
+    }
+
+    /**
+     * Job class which contains the task to be run periodically by the scheduler
+     */
```

Comment:
This should actually explain what is the Job about ... so deleting the inactive consumers.